setup.py

```diff
@@ -69,30 +66,55 @@ def exit_with_error(message):
         CFLAGS = arg.split('=')[1].split()
         sys.argv.remove(arg)
 
+
 # Blosc sources and headers
+
+# To avoid potential namespace collisions use build_clib.py for each codec
+# instead of co-compiling all sources files in one setuptools.Extension object.
+clibs = [] # for build_clib, libraries TO BE BUILT
+
+# Below are parameters for the Extension object
 sources = ["blosc/blosc_extension.c"]
 inc_dirs = []
 lib_dirs = []
-libs = []
+libs = []  # Pre-built libraries ONLY, like python36.so
 def_macros = []
 if BLOSC_DIR != '':
     # Using the Blosc library
     lib_dirs += [os.path.join(BLOSC_DIR, 'lib')]
     inc_dirs += [os.path.join(BLOSC_DIR, 'include')]
     libs += ['blosc']
 else:
+    # Libraries to be built with build_clib
+    clibs.append( ('lz4', {'sources': glob('c-blosc/internal-complibs/lz4*/*.c')} ) )
+
+    # Tried and failed to compile Snappy with gcc using 'cflags' on posix
+    # setuptools always uses gcc instead of g++, as it only checks for the 
+    # env var 'CC' and not 'CXX'.
+    clibs.append( ('snappy', {'sources': glob('c-blosc/internal-complibs/snappy*/*.cc'), 
+                               'cflags':'-std=c++11 -lstdc++' } ) )
```

Comment:
So, if Snappy is not possible to be compiled with gcc I propose to disable it permanently from python-blosc.  Could you comment these lines out?

Reply:
Another possibility is to honor some environment variable so that Snappy can be compiled in.  Perhaps that would be a good trade off.